app.py

```diff
@@ -1,24 +1,24 @@
-from datetime import datetime
 import os
+from datetime import datetime
 
-from flask import Flask, render_template, request, redirect, url_for, send_from_directory
-from flask_sqlalchemy import SQLAlchemy
+from flask import Flask, redirect, render_template, request, send_from_directory, url_for
 from flask_migrate import Migrate
+from flask_sqlalchemy import SQLAlchemy
 from flask_wtf.csrf import CSRFProtect
 
 
 app = Flask(__name__, static_folder='static')
 csrf = CSRFProtect(app)
 
 # WEBSITE_HOSTNAME exists only in production environment
-if not 'WEBSITE_HOSTNAME' in os.environ:
-   # local development, where we'll use environment variables
-   print("Loading config.development and environment variables from .env file.")
-   app.config.from_object('azureproject.development')
+if 'WEBSITE_HOSTNAME' not in os.environ:
```

Comment:
Ruff recommends "not in" versus "not X in" due to it resulting in different true/false values in some situations, I believe.